kytos/of_lldp/main.py

```diff
@@ -78,19 +78,27 @@ def update_links(self, event):
             lldp.unpack(ethernet.data.value)
 
             dpid = DPID()
-            dpid.unpack(lldp.chassis_id.sub_value.value)
+            if lldp.chassis_id.sub_value.value != b'':
```

Comment:
on napps we should avoid to use binary format and .value. But ok, we can fix this latter